cli.go

```diff
@@ -19,8 +28,18 @@ func main() {
 	var installOpenAsar = flag.Bool("install-openasar", false, "Install OpenAsar on a Discord install")
 	var uninstallOpenAsar = flag.Bool("uninstall-openasar", false, "Uninstall OpenAsar from a Discord install")
 	var updateFlag = flag.Bool("update", false, "Update your local Vencord files")
+	var dir = flag.String("dir", "", "Select the location of your Discord client")
+	var client = flag.String("client", "", "Select the branch of Discord you wish to modify [default|stable|ptb|canary]")
 	flag.Parse()
 
+	if *dir != "" && *client != "" {
+		log.Fatal("The 'dir' and 'client' flags are mutually exclusive.")
```

Comment:
please change these usages of the `log` package to `fmt`. log adds pointless timestamps which distract from the actually relevant information
![image](https://github.com/Vencord/Installer/assets/45497981/d7ed71e1-0d64-4ff0-aa68-557edd733a8f)